Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ChannelGroup/ByChannelGroup.tsx

```diff
@@ -133,29 +133,28 @@ const ChannelGroups: Application.Types.iByComponent = (props) => {
                     Selected={(item) => false}
                     KeySelector={(item) => item.ID}
                 >
-                    <ReactTable.Column<SystemCenter.Types.ChannelGroup>
+                    <ReactTable.Column<LocalSC.ChannelGroupView>
                         Key={'Name'}
                         AllowSort={true}
                         Field={'Name'}
                         HeaderStyle={{ width: '15%' }}
                         RowStyle={{ width: '15%' }}
                     > Name
                     </ReactTable.Column>
-                    <ReactTable.Column<SystemCenter.Types.ChannelGroup>
+                    <ReactTable.Column<LocalSC.ChannelGroupView>
                         Key={'Description'}
                         AllowSort={true}
                         Field={'Description'}
                         HeaderStyle={{ width: 'auto' }}
                         RowStyle={{ width: 'auto' }}
                     > Description
                     </ReactTable.Column>
-                    <ReactTable.Column<SystemCenter.Types.ChannelGroup>
+                    <ReactTable.Column<LocalSC.ChannelGroupView>
                         Key={'Items'}
```

Comment:
```suggestion
                        Key={'ItemCount'}
```
Arrow for sort indication is based on key not field, so key must also be ItemCount